docs/user-guide/ze-cics-usage-tips.md

```diff
@@ -0,0 +1,9 @@
+# Usage tips
+
+The following tips can help you perform tasks more efficiently when working with the CICS extension:
+
+- All menu action commands available by right-clicking a profile or resource (excluding **Show Attributes**) can be applied to multiple items. To do this, select the multiple nodes of the same type before right-clicking and selecting the command.
+
+- To select multiple nodes, you can hold Ctrl/Cmd key while clicking resources. Or you can select the first item in a list of nodes then hold Shift and click both the last item to select a consecutive list of nodes.
```

Comment:
To select multiple nodes, hold the Ctrl/Cmd key while clicking on the resources. You can also select multiple consecutive nodes by selecting the first item in a list of nodes and then holding the Shift key while selecting the last item in the list.